src/stylesheets/components/_navigation.scss

```diff
@@ -62,11 +63,14 @@
     overflow-y: visible;
     position: relative;
     width: auto;
+    visibility: visible;
   }
 
   &.is-visible {
     @include transform(translateX(0));
-    @include transition(all 0.3s ease-in-out);
+    @include transition(transform 0.3s ease-in-out);
```

Comment:
Changing the transition from `all` to just `transform` was crucial for ensuring that we could still focus on the close button immediately after the user clicked on the menu button: with transition set to `all`, even `visibility` was being transitioned, which meant that when `focus()` was called on the close button, nothing happened because the button wasn't actually visible yet!